apps/angular/crud/src/styles.scss

```diff
@@ -8,3 +8,15 @@ body {
   margin: 0;
   font-family: Roboto, 'Helvetica Neue', sans-serif;
 }
+
+.d-block {
```

Comment:
this project utilize [tailwindcss](https://tailwindcss.com/docs/margin), just use `block` instead of `d-block`, `m-auto` for `margin-auto`, ..., then you don't need to declare these file at all -> PR cleaner 🚀

Reply:
hmm then maybe try to add `@tailwind utilities;` to this file

Reply:
Added some basic tailwind utilities, ty for the feedback never worked previously with tailwind but looks really good.